client/lib/analytics/ad-tracking.js

```diff
@@ -511,7 +512,9 @@ function recordProduct( product, orderId ) {
 		// Google AdWords
 		if ( window.google_trackConversion ) {
 			window.google_trackConversion( {
-				google_conversion_id: GOOGLE_CONVERSION_ID,
+				google_conversion_id: isJetpackPlan
```

Comment:
I think your logic is backwards. `GOOGLE_CONVERSION_ID_JETPACK` should probably come after the `?` in the ternary, right?